CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Changelog
 
+## 4.2.2
+
+* Fix `to_value_json` for `DatedVariable` with extra parameters.
+
+This was causing a crash when calculating intermediate variables with the API.
+
 ## 4.2.1
 
-  Fix permanent and period size independent variables neutralization
+* Fix permanent and period size independent variables neutralization
```

Comment:
But it works with both `-` and `*`, so we will always use the wrong one... Why not accept both?

GitHub [mastering Markdown](https://guides.github.com/features/mastering-markdown/) howto shows we can use both (in examples/list section).

On the other hand, having both creates an impression of low quality :-)

@MattiSG Any thoughts?